README.md

```diff
@@ -92,8 +92,8 @@ Due to our use of `cgo`, you'll need a C compiler to build go-filecoin whether y
 `go-filecoin` depends on some proofs code written in Rust, housed in the
 [rust-fil-proofs](https://github.com/filecoin-project/rust-fil-proofs) repo and consumed as a submodule. You will need to have `cargo` and `jq` installed.
 
-go-filecoin's dependencies are managed by [gx][2]; this project is not "go gettable." To install gx, golangci-lint, and
-other build and test dependencies (with precompiled proofs, recommended), run:
+To install gx, golangci-lint, and other build and test dependencies (with
```

Comment:
Is gx still a build and test dependency after this PR?

Reply:
No. Thanks for catching.